src/spec/features/password_resets_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'spec_helper'
+
+feature 'Password reset' do
+  let(:user) { create(:participant, name: 'Hedy Lamarr', email: 'freq-hopper@example.org') }
+
+  scenario 'The home page has a link to the reset password page' do
+    visit root_path
+    assert page.has_link?('Forgot Password?', href: new_password_reset_path)
+  end
+
+  scenario 'The reset password page has the expected content' do
+    visit new_password_reset_path
+    assert page.has_css?('h1', text: 'Reset Password')
+    assert page.has_text?('Please enter your email address below')
+  end
+
+  scenario 'Participant can reset password' do
+    # Event needed until Pull Request #202 is merged into the code base
+    Event.create!(name: 'Event 1', date: Time.now)
+
+    # Enter an invalid email address
+    visit new_password_reset_path
+    fill_in 'email', with: 'not_exist@example.com'
+    click_button 'Reset Password'
+    assert page.has_text?('No participant was found with email address not_exist@example.com')
+
+    # Enter a valid email address
+    visit new_password_reset_path
+    fill_in 'email', with: user.email
+    click_button 'Reset Password'
+    assert page.has_text?('Instructions to reset your password have been emailed to you')
+
+    # Open and follow instructions
+    open_email(user.email)
+    assert current_email.subject.include?('Password Reset Instructions')
+    assert current_email.body.include?('A request to reset your password has been made.')
+    current_email.click_link 'Reset Password!'
+    clear_emails
+
+    # Provide new password
+    assert page.has_css?('h1', text: 'Update your password')
+    assert page.has_text?('Please enter the new password below')
+    fill_in('password', with: 'MN Tech Community')
+    sleep 4.1
```

Comment:
We'll want to remove this. If you specify a `find('.some-selector.or-whatever', wait: 4)` you can specify a default max wait time. You might want to check but it might already be 5 or more seconds.

Reply:
I believe this is the only substantial change requested. I'd be happy to merge this if we can remove the sleep call.